src/tests/ftest/util/server_utils.py

```diff
@@ -107,8 +91,11 @@ def create_server_yaml(basepath):
             elif key in default_value_set:
                 default_value_set[key] = new_value_set['server_config'][key]
 
-    #Disable NVMe from baseline data/daos_server_baseline.yml
-    set_nvme_mode(default_value_set, "nvme")
+            if key in default_value_set.keys():
```

Comment:
Isn't this just a repeat of lines 88-92?

Reply:
Yes it is duplicate. will remove one set.